src/kaltura-player.js

```diff
@@ -77,4 +79,16 @@ export default class KalturaPlayer {
       set: undefined
     };
   }
+
+  _addBindings(): void {
```

Comment:
This is massive and too specific referring between kaltura player and a plugin
If the problem is that the plugin cannot listen to the UI events the solution is to move the plugin manager to the kaltura player level and not bind the kaltura player to a specific plugin

Reply:
one more thing is - if we will run kaltura player in chromeless mode (`{ui: {disable: true}}`) the vr navigation won't work.

Reply:
@dan-ziv, @yairans  we will move this in to the player and then the UI will call `player.toggleStereoMode()` directly

Reply:
done